Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,241 @@
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Dev              	| Unstable bits that are "bleeding edge". Users are nto expected to us this channel often, however it is there for those situations when someone needs a feature or maybe a new bug fix urgently that hasn't been stabilizied yet. Also, used for internal testing. 	|
+| Beta/Pre-release 	| Pre-release stable builds with known issues and/or known feature gaps. We are OK with users installing these for preview purposes.                                                                                                                                	|
+| Release          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+Channels also impact the NuGet packages. Based on the version of the package specified, the user may end up getting different channels. This is described in more details in [Nuget Packages section](#nuget-packages).  
+
+# Dependencies
+.NET Core CLI is built on top of CoreFX and CoreCLR and as such its' dependencies set is defined by the platform that those two combine. Some of the install options provide an automatic way to install these dependencies, while other do not. However, it is important to note that the CLI bundle will **not carry those dependencies with it** in any case. This means that for those install options that do not support automatic 
+
+A list of dependencies can be found on [](). 
+
+# Upgrades
+**TODO**
+
+# Layout on disk
+**TODO**
+
+# Acquisition vectors
+By "vectors" we are refering here to the way any given user can obtain the installers. In the CLI, there are two main planned vectors:
+
+1. [Getting Started Page](https://aka.ms/dotnetcoregs)
+2. [Repo landing page](https://github.com/dotnet/cli/blob/rel/1.0.0/README.md)
+
+There are two more planned vectors:
+
+1. Visual Studio
+2. VS Code 
+
+## Getting Started page
+The page can be found on https://aka.ms/dotnetcoregs. This is the main curated first-run experience for the dotnet CLI. The intent of the page is to help users "kick the tires" quickly and become familiar with what the platform offers. This should be the most stable and curated experience we can offer. Getting started page can never point to unstable builds.
```

Comment:
general comment: install is a verb. I think we should be saying installation and Installer